### DISCUSSION TITLE
WIN11安全中心报毒

### DISCUSSION THREAD

**Discussion by msxdro** [shuax/MouseInc.Settings#387] [General]
升级完2.13.2就报
![Snipaste_2022-09-08_15-01-42](https://user-images.githubusercontent.com/113091350/189057987-585f7aab-2adc-46a2-8b7b-381e608bbd61.jpg)

**Comment by shuax** [comment]
信则用，不信不用

**Reply by msxdro** [reply]
我信，但别人也一样会碰到吧，是什么技术原因吗？

**Comment by shuax** [comment]
遇到就遇到呗，去找微软问。
